app/assets/stylesheets/components/team_members.scss

```diff
@@ -1,3 +1,9 @@
 .small-table-button {
   margin-bottom: 0px;
 }
+
+@media only screen and (max-width: 400px) {
+  #pending-invite {
+    display: none;
```

Comment:
Won't this hide the element on small screens? Is that the desired behaviour?

From the PR/commit titles, I'd guessed this would shrink or move the text rather than hide it.